### PR TITLE
Glass UI v1

### PR DESCRIPTION
**Pull request by joshua-s**
Improvements to the light theme

### CODE REVIEW

File: laskyawm.css [hunk 1]
```diff
@@ -31,8 +31,8 @@ body:before {
 	position: absolute !important;
 	top: 25px;
 	left: 100px;
-	box-shadow: 0px 0px 20px rgba(0, 0, 0, .3);
 	background-color: rgba(255, 255, 255, .2);
+	border: 1px solid transparent;
```
Comment (twiss) [inline laskyawm.css]: I have to admit the white border looks pretty cool. And I also agree that the black box-shadow doesn't fit very well with the light theme. However, without it it's kinda hard to see where one window ends and another starts, especially when everything is white (Firetext is white / light grey, Marketplace is white, border is now white). One idea would be to make the border the color of the (window) background, e.g. `.window .tabs { border: 1px solid rgba(255, 255, 255, .5); border-top: 0; }` but it doesn't work that well in Chrome. Other ideas?

Comment (joshua-s) [inline laskyawm.css]: Did you see the slight accented border:
untitled

Comment (joshua-s) [inline laskyawm.css]: Oh, that only works when they are not overlapping. Hmm

Comment (joshua-s) [inline laskyawm.css]: I added a light box shadow. Check it out!

**Comment by joshua-s** [comment]
In the recent update I added window animation.

**Comment by twiss** [comment]
These animations are really cool. However, on my computer the animations that change the width and height of a window (so (un)maximize and (un)minimize) are not fluent. Maybe, you could set the width and height directly on the iframe as well, so that the size of the iframe is not animated, but the size of the window is.

Also, this seems to cause bugs with force-minimizing covered windows, and also with the placement of resizable handles. Maybe, it would help to set width and height properties on the `div`/`win` object and make a function to return the real width/height of a window based on those properties and whether or not the window is maximized (kinda like this: https://github.com/airbornos/airbornos/blob/986adab/laskyawm.js#L538-577).

Lastly, could you separate the animation stuff and the style changes into two PRs? I'm still kind of doubtful about the style changes. For example, the text shadow is there so that it's still easily readable when the background is white or really light. Now, the background generator never makes the background completely white except in the center. However, it can be light such as this one: 
screenshot-text-shadow
screenshot-no-text-shadow

**Comment by joshua-s** [comment]
I can do all of that. Here's one question: could we prevent the light issue either by inverting for bright backgrounds or not allowing the generator to produce past a certain threshold?

**Comment by twiss** [comment]
Yeah, that could work, although if we ever allow people to upload their own backgrounds it's possible to come up with black and white backgrounds where that still wouldn't work.

**Pull request closed without merging**